.github/workflows/snyk-sbom.yml

```diff
@@ -0,0 +1,25 @@
+name: Snyk - Software Bill of Materials (SBOM)
+on:
+  release:
+    types: [published]
```

Comment:
Releases are being executed from a different repository, and release workflows are private. 
Not sure if this event will be processed here. We need to research this.

Reply:
It will I think . https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release 🚀  
Nice thought @enzowritescode